packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -190,28 +190,33 @@ class BlobOnlyStorage implements IDocumentStorageService {
 	}
 
 	public get policies(): IDocumentStorageServicePolicies | undefined {
-		return this.notCalled();
+		return this.notCalled("policies");
 	}
 
 	public get repositoryUrl(): string {
-		return this.notCalled();
+		return this.notCalled("repositoryUrl");
 	}
 
 	/* eslint-disable @typescript-eslint/unbound-method */
-	public getSnapshotTree: () => Promise<ISnapshotTree | null> = this.notCalled;
-	public getVersions: () => Promise<IVersion[]> = this.notCalled;
-	public write: () => Promise<IVersion> = this.notCalled;
-	public uploadSummaryWithContext: () => Promise<string> = this.notCalled;
-	public downloadSummary: () => Promise<ISummaryTree> = this.notCalled;
+	public getSnapshotTree: () => Promise<ISnapshotTree | null> = () =>
```

Comment:
I understand the convenience of having this in the event as a property, but in the interest of keeping the bundle size low and maybe the telemetry database size, don't we already have this information in the stack trace?

Something like:
```
union Office_Fluid_FluidRuntime_*
| where Event_Time > ago(1d)
| where Data_eventName == "fluid:telemetry:Container:BlobOnlyStorageWrongCall"
| summarize count() by Data_stack
| order by count_ desc 
| limit 10
```
will likely get you the most relevant scenarios to investigate.